Assets/Scripts/Player/PlayerController.cs

```diff
@@ -259,7 +289,7 @@ public class PlayerController : MonoBehaviour {
 					crawlingSound.Play();
 			}
 			else {
-				crawlingSound.Stop();
+				crawlingSound.Pause();
```

Comment:
This can probably be simplified like the other sounds. You should also move this sound handling above the return on 249 since the sound doesn't stop on transformation start.